cloudferrylib/utils/drivers/ssh_chunks.py

```diff
@@ -187,3 +215,40 @@ def transfer(self, data):
                 LOG.error(message)
                 remote_rm_file(dst_runner, dst_path)
                 raise FileCopyFailure(message)
+
+
+class RsyncCopier(driver_transporter.DriverTransporter):
+    """Uses `rsync` to copy files. Used by ephemeral drive copy process"""
+
+    def transfer(self, data):
+        src_host = data['host_src']
+        src_path = data['path_src']
+        dst_host = data['host_dst']
+        dst_path = data['path_dst']
+
+        src_user = self.cfg.src.ssh_user
+        dst_user = self.cfg.dst.ssh_user
+        src_password = self.cfg.src.ssh_sudo_password
+
+        src_runner = remote_runner.RemoteRunner(src_host,
+                                                src_user,
+                                                password=src_password,
+                                                sudo=True)
+
+        rsync = ("rsync "
+                 "--perms "
+                 "--owner "
+                 "--group "
+                 "--compress "
+                 "--verbose "
+                 "--progress "
+                 "--partial "
+                 "--rsh='ssh -o StrictHostKeyChecking=no' "
```

Comment:
Consider also adding `UserKnownHostsFile=/dev/null` since ssh could fail due to different host key in `~/.ssh/known_hosts`. Please also merge patch that adds ability to choose cipher and add cipher selection functionality here.